frontend/src/concepts/k8s/NameDescriptionField.tsx

```diff
@@ -22,6 +28,8 @@ const NameDescriptionField: React.FC<NameDescriptionFieldProps> = ({
   autoFocusName,
   showK8sName,
   disableK8sName,
+  isNameValid,
```

Comment:
Since this is a generic component, not every usage supplies a value. You'll want to supply a default value for the cases where no validation is happening.
```suggestion
  isNameValid = true,
```